e2e-tests/IIoTPlatform-E2E-Tests/deploy/IIotHubDeployment.cs

```diff
@@ -0,0 +1,22 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace IIoTPlatform_E2E_Tests.TestExtensions {
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    public interface IIoTHubDeployment {
```

Comment:
@dacolgit, GitHub shows that two files are added with seemingly the same content:

* `e2e-tests/IIoTPlatform-E2E-Tests/deploy/IIoTHubDeployment.cs`
* `e2e-tests/IIoTPlatform-E2E-Tests/deploy/IIotHubDeployment.cs `

Please remove the second one as it has lowercase t in `IoT`.

Reply:
removed